### PR TITLE
[EDU-2571] Add L2 region info

### PR DESCRIPTION
**Pull request by bru-andrade**
### Changes

- Modified default region info
- Added early access note

[EDU-2571]: https://aziontech.atlassian.net/browse/EDU-2571?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/products/edge-application/l2-caching.mdx [hunk 1]
```diff
@@ -15,9 +20,7 @@ When accessing your applications on Azion, your end-user accesses the edge netwo
 - Keep your content cached for longer periods of time.
 - Select the best region to keep cache closer to your users.
 
-:::note
-L2 Caching is only available if the service has been activated for your account. To activate **L2 Caching**, [contact the Sales team](https://www.azion.com/en/contact-sales/).
-:::
+To activate **L2 Caching**, [contact the Sales team](https://www.azion.com/en/contact-sales/).
```
Comment (bru-andrade) [inline src/content/docs/en/pages/products/edge-application/l2-caching.mdx:23]:
```suggestion
:::note
L2 Caching is only available if the service has been activated for your account. To activate **L2 Caching**, [contact the Sales team](https://www.azion.com/en/contact-sales/).
:::
```

File: src/content/docs/pt-br/pages/produtos/edge-application/l2-caching/index.mdx [hunk 2]
```diff
@@ -38,10 +43,9 @@ L2 Caching só estará disponível se o serviço for ativado para a sua conta. P
 
 ## Regiões
 
-Ao usar o **cache L2** como uma camada de cache adicional, você deve selecionar uma região de servidor. As regiões disponíveis são:
+Quando você ativa o **L2 Caching** como uma camada extra de cache, os servidores edge dessa segunda camada de cache são hospedados nos EUA (`na-united-states`) por padrão. 
 
-- Brasil: `sa-brazil`
-- EUA: `na-united-states`
+A Azion também oferece a opção de hospedar os servidores L2 no Brasil (`sa-brazil`). Para alternar entre as regiões em suas aplicações de borda, entre em contato com a [equipe de vendas](https://www.azion.com/en/contact-sales/).
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/produtos/edge-application/l2-caching/index.mdx:48]:
```suggestion
A Azion também oferece a opção de hospedar os servidores L2 no Brasil (`sa-brazil`). Para alternar entre as regiões em suas edge applications, entre em contato com a [equipe de vendas](https://www.azion.com/en/contact-sales/).
```

**Review by MarianaAguilera — APPROVED** [review]

File: src/content/docs/en/pages/products/edge-application/l2-caching.mdx [hunk 3]
```diff
@@ -6,6 +6,11 @@ namespace: documentation_products_edge_application_l2_caching
 permalink: /documentation/products/edge-application/l2-caching/
 ---
 import Button from '~/components/Button.astro';
+import Badge from '~/components/Badge.astro';
+
+<Badge variant="accent">
+Early Access
```
Comment (hannah-sk) [inline src/content/docs/en/pages/products/edge-application/l2-caching.mdx:12]:
```suggestion
Preview
```

File: src/content/docs/pt-br/pages/produtos/edge-application/l2-caching/index.mdx [hunk 4]
```diff
@@ -6,6 +6,11 @@ namespace: documentation_products_edge_application_l2_caching
 permalink: /documentacao/produtos/edge-application/l2-caching/
 ---
 import Button from '~/components/Button.astro';
+import Badge from '~/components/Badge.astro';
+
+<Badge variant="accent">
+Early Access
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/produtos/edge-application/l2-caching/index.mdx:12]:
```suggestion
Preview
```

File: src/content/docs/en/pages/products/edge-application/l2-caching.mdx (hunk 3 above)
Comment (hannah-sk) [reply src/content/docs/en/pages/products/edge-application/l2-caching.mdx:12]: I believe that's the current equivalent

**Review by hannah-sk — APPROVED** [review]
left a few corrections

File: src/content/docs/en/pages/products/edge-application/l2-caching.mdx [hunk 5]
```diff
@@ -6,6 +6,11 @@ namespace: documentation_products_edge_application_l2_caching
...
+</Badge>
```
Comment (bru-andrade) [inline src/content/docs/en/pages/products/edge-application/l2-caching.mdx:13]:
```suggestion
```

File: src/content/docs/pt-br/pages/produtos/edge-application/l2-caching/index.mdx [hunk 6]
```diff
@@ -6,6 +6,11 @@ namespace: documentation_products_edge_application_l2_caching
...
+</Badge>
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/produtos/edge-application/l2-caching/index.mdx:13]:
```suggestion

```

**Pull request merged**